### DISCUSSION TITLE
OCR on Android

### DISCUSSION THREAD

**Discussion by renanpaivamobi** [Strypper/mauisland#200] [Q&A] [answered]
Hi everyone,

Does the OCR work on Android mobile devices?

**Comment by Strypper** [comment, marked as answer]
We haven't had the opportunity to start an Android project yet. We'll try one and update you this Sunday. Is that okay

**Comment by Strypper** [comment]
Sorry for the late response I can confirm it work on Android, but for IOS our team still doesn't have a Mac or Apple device to test it out @renanpaivamobi
